src/distributors/base-installer.ts

```diff
@@ -15,6 +15,7 @@ export abstract class JavaBase {
   protected version: semver.Range;
   protected arch: string;
   protected javaPackage: string;
+  protected access: string;
```

Comment:
Let's rename `access` to `stable` (true, false)

Reply:
Done. Thank you.